service/controller/flannel.go

```diff
@@ -14,7 +14,9 @@ import (
 	"github.com/giantswarm/flannel-operator/service/controller/v3"
 )
 
-type FrameworkConfig struct {
+const flannelControllerSuffix = ""
```

Comment:
Should be `"-flannel"` eventually.

Reply:
IMO we should go with the plain operator name if there is only one controller.